Samples/Tutorial Samples/WinMLPIXSample/WinMLPIXSample/readme.txt

```diff
@@ -0,0 +1,10 @@
+========================================================================
```

Comment:
Can this me a .md file? That way people who are browsing code on the github repo can see this rendered whenever they navigate to this folder.

Reply:
Sure.